src/Applications/LDAS_App/process_rst.csh

```diff
@@ -89,9 +92,8 @@ mpirun -map-by core --mca btl ^vader -np 56 bin/mk_LDASsaRestarts -a ${SPONSORID
 
 _EOI_
 
-    if($LSM_CHOICE == 1) sed -i '$ a\bin/Scale_Catch OutData1/catch_internal_rst OutData2/catch_internal_rst catch_internal_rst 50 \'  mkLDASsa.j
-    if($LSM_CHOICE == 2) sed -i '$ a\bin/Scale_CatchCN OutData1/catchcn_internal_rst OutData2/catchcn_internal_rst catchcn_internal_rst 50 \'  mkLDASsa.j
-    #if($LSM_CHOICE == 2) sed -i '$ a\bin/Scale_Catch OutData1/catchcn_internal_clm45 OutData2/catchcn_internal_clm45 catchcn_internal_clm45 50 \'  mkLDASsa.j
+    if($LSM_CHOICE == 1) sed -i "$ a\bin/Scale_Catch OutData1/catch_internal_rst OutData2/catch_internal_rst catch_internal_rst 50 $WEMIN_IN $WEMIN_OUT \"  mkLDASsa.j
```

Comment:
should we be hardcoding surflayer here?

Reply:
Good question ! In AGCM, Ganymed-4_0 and later use 50
It is not an optional user input argument in regrid.pl, though.

This is more important than WEMIN, because surflay points to different columns in tau_param.dat, (model parameters) and those parameters go to restart files.

Reply:
now fixed with be312a3